packages/material-ui/src/styles/createSpacing.d.ts

```diff
@@ -1,4 +1,19 @@
-export type Spacing = (value: number | string) => number | string;
+/* tslint:disable:unified-signatures */
```

Comment:
For future reference: The reason for disabling this rule here is that `tslint` suggests using `(value1: SpacingArgument, value2?: SpacingArgument)`. This would however accept `spacing(1, undefined)` which we do not want to accept.